ios/Plugin/Plugin.swift

```diff
@@ -68,6 +66,17 @@ public class RadarPlugin: CAPPlugin, RadarDelegate {
         }
     }
 
+    override public func load() {
+        // By default, Capacitor passes JavaScript Dates to native code as
+        // ISO8601 strings. Setting this to false keeps them as Date objects,
+        // so no further string -> Date parsing needs to take place in iOS.
+        // (Android doesn't have this property, so it _does_ need to do the
```

Comment:
Why do you think Android doesn't support this?

Reply:
Android's `Plugin` doesn't have this property, nor does it have the `getDate()` that the docs mention.